### PR TITLE
Update main page and images to match current name

### PR DESCRIPTION
**Pull request by Equalizer5118**
Updated name usage to strata

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -2,26 +2,26 @@
...
+description: "Strata Source engine documentation"
 permalink: /
 ---
 
-# Chaos Engine Wiki
+# Strata Source engine wiki
 
-Welcome to the Chaos engine wiki! This wiki contains documentation that is
-common to all Chaos-based games.
+Welcome to the Strata Sourceengine wiki! This wiki contains documentation that is
```
Comment (tsa96) [inline index.md:11]: Missing space - "Sourceengine"

**Review by tsa96 — CHANGES_REQUESTED** [review]

File: index.md (hunk 1 above)
Comment (Equalizer5118) [reply index.md]: Aight, fixed. Thanks man.

**Review by tsa96 — CHANGES_REQUESTED** [review]
You should use the .svg version of the favicon at well, that one adapts to whether the user is using light or dark mode

**Comment by Equalizer5118** [comment]
Alright, removed old ico and added svg instead. What do you think?

**Comment by tsa96** [comment]
Not sure where you got that SVG from, the dynamic SVG is https://github.com/StrataSource/strata-branding/blob/main/i/favicon.svg

Sorry if hard to find, not really sure what's going on with the Strata branding tool tbh. But that one I linked should be used for all favicons.

You should still include an ICO favicon though, for browsers that don't support SVGs.

**Comment by tsa96** [comment]
And you'll need to actually update the HTML to add it. Maybe worth pinging me on someone else in Discord if you're not familiar with HTML

**Comment by Equalizer5118** [comment]
Alright, I'll do that. I'm still learning HTML and GitHub, so I apologize for my inability to do this efficiently.

**Review by tsa96 — APPROVED** [review]

**Comment by tsa96** [comment]
Looks okay, though I don't have a local setup to test this. JJ can you check it out, or just squash (def worth squashing, commit history is gross) it if you're happy?

**Comment by Equalizer5118** [comment]
I tested this locally and everything checks out on my end

**Review by JJL772 — APPROVED** [review]
Oh whoops, didn't see the "pls review" email from github!

Looks good to me, thanks for updating this for us!

**Pull request merged**